rollup.config.cjs

```
@@ -0,0 +1,100 @@
+    output: [
+      {
+        file: packageJson.main,
+        format: 'esm',
+        sourcemap: true,
+      },
+    ],
+    plugins: [
+      external(),
+      alias({
+        entries: [
+          { find: '@', replacement: path.resolve(__dirname, 'src') }
```

ESLint

[error] '__dirname' is not defined.

(no-undef)